util/cpu/cpu_test.go

```diff
@@ -23,11 +23,15 @@ import (
 	"github.com/pingcap/failpoint"
 	"github.com/pingcap/tidb/resourcemanager/scheduler"
 	"github.com/pingcap/tidb/resourcemanager/util"
+	"github.com/pingcap/tidb/util/cgroup"
 	"github.com/pingcap/tidb/util/cpu"
 	"github.com/stretchr/testify/require"
 )
 
 func TestCPUValue(t *testing.T) {
+	if !cgroup.InContainer() {
+		t.Skip()
```

Comment:
Could you please add a comment here?

Reply:
Or we can set a message for this skip, like t.Skip("xxx").

Reply:
Done